doc/how_to/load_matalb_data.rst

```diff
@@ -0,0 +1,66 @@
+Exporting MATLAB Data to Binary & Loading in SpikeInterface
+===========================================================
+
+In this tutorial, we'll go through the process of exporting your data from MATLAB in a binary format and then loading it using SpikeInterface in Python. Let's break down the steps.
+
+Exporting Data from MATLAB
+--------------------------
+
+First, ensure your data is structured correctly. The data matrix should be organized such that the first dimension corresponds to samples/time and the second dimension to channels.
+
+.. code-block:: matlab
+
+   % Define the size of your data
+   num_samples = 1000;
+   num_channels = 384;
```

Comment:
Tiny point, and not official, but most Matlab users use camel case instead of snake case. As you do below.

Reply:
I think it's a very good point and it makes the Python/MATLAB distinction even clearer :)

Reply:
Yes, thanks. I was concerned as well that because of the format it will not be clear that we talking about matlab code. I switched to camelCase and added an extra sentence to clarify this is indeed matlab code.